nipy/labs/spatial_models/parcel_io.py

```diff
@@ -100,7 +101,8 @@ def parcel_input(mask_images, learning_images, ths=.5, fdim=None):
         # mask_images should be a list
         grp_mask = intersect_masks(mask_images, threshold=ths) > 0
         mask = Nifti1Image(grp_mask.astype('u8'),
-                           get_affine(load(mask_images[0])))
+                           get_affine(load(mask_images[0])),
+                           dtype=grp_mask.astype('u8').dtype)
```

Comment:
I suspect this was meant to be `uint8` (`u1`), and instead it's `uint64` (`u8`). Why would you insist on creating a mask of 64-bit ints?